ChangeLog.d/updated_windows_apis.txt

```diff
@@ -0,0 +1,5 @@
+API changes
```

Comment:
Isn't this section for changes to the Mbed TLS API, rather than the APIs it calls?

Reply:
Yes. This is a new feature.

Reply:
Although if.this drops support for older Windows, which seems to be the case, then that should also be documented under “Platform requirement changes”.